### PR TITLE
export csv button position

### LINKED ISSUE #1589: CSV button Download not aligned
Screenshot 2022-10-06 at 09 56 19

### PR DESCRIPTION
**Pull request by rfontanarosa**
close #1589

Schermata 2022-10-13 alle 13 45 32

### CODE REVIEW

File: src/client/components/EditUserForm/CountryRoles/CountryRoles.tsx [hunk 1]
```diff
@@ -87,7 +89,9 @@ const CountryRoles: React.FC<{ user: User }> = ({ user }) => {
   return (
     <div className="edit-user__form-item edit-user__form-item-roles">
       <div className="edit-user__form-label">{i18n.t<string>('editUser.role')}</div>
-      <div className={`edit-user__form-field edit-user__form-field-roles${Users.validRole(user) ? '' : ' error'}`}>
+      <div
+        className={classNames('edit-user__form-field', 'edit-user__form-field-roles', { error: Users.validRole(user) })}
```
Comment (sorja) [inline src/client/components/EditUserForm/CountryRoles/CountryRoles.tsx]: Should this be `!Users.validRole`

File: src/client/components/Navigation/NavAssessment/Header/Status/StatusConfirm.tsx [hunk 2]
```diff
@@ -57,7 +60,7 @@ const StatusConfirm: React.FC<Props> = (props) => {
             role="button"
             tabIndex={0}
           >
-            <div className={`fra-checkbox${notifyUsers ? '' : ' checked'}`} />
+            <div className={classNames('fra-checkbox', { checked: notifyUsers })} />
```
Comment (sorja) [inline src/client/components/Navigation/NavAssessment/Header/Status/StatusConfirm.tsx]: should this be `!notifyUsers`

Comment (minotogna) [reply src/client/components/Navigation/NavAssessment/Header/Status/StatusConfirm.tsx]: Wasn't that in the other pr ?

Comment (rfontanarosa) [reply src/client/components/Navigation/NavAssessment/Header/Status/StatusConfirm.tsx]: the only one with inverted logic... :(

Comment (rfontanarosa) [reply src/client/components/Navigation/NavAssessment/Header/Status/StatusConfirm.tsx]: yes mino, it's in both

**Review by sorja — APPROVED** [review]

**Review by minotogna — APPROVED** [review]

**Pull request merged**